src/plugin/CheckoutBillingContainer.plugin.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * ScandiPWA - Progressive Web App for Magento
+ *
+ * Copyright © Scandiweb, Inc. All rights reserved.
+ * See LICENSE for license details.
+ *
+ * @license OSL-3.0 (Open Software License ("OSL") v. 3.0)
+ * @package scandipwa/braintree-graphql
+ * @link https://github.com/scandipwa/braintree-graphql
+ */
+
+import { BRAINTREE } from '../component/Braintree/Braintree.config';
+
+export class CheckoutBillingContainerPlugin {
+    onPaymentSavedInVaultChange() {
+        this.setState(({ isSavePayment }) => ({ isSavePayment: !isSavePayment }));
+    }
+
+    aroundContainerFunctions = (originalMember, instance) => ({
+        ...originalMember,
+        onPaymentSavedInVaultChange: this.onPaymentSavedInVaultChange.bind(instance)
+    });
+
+    aroundGetPaymentData(args, callback, instance) {
+        const { paymentMethod: code, isSavePayment } = instance.state;
+
+        if (code === BRAINTREE) {
+            const [{ 0: { nonce } }] = args;
```

Comment:
Please don't use `{ 0: ... }` to destructure arrays. Is it possible to use `[[{nonce}]]` ?